### PR TITLE
Local Test Environment

### PR DESCRIPTION
**Pull request by pcgames3112**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -31,9 +31,10 @@
  * We can watch errors happening live. Grepping them can also help
  * filtering.
  */
-ini_set("log_errors" , "1");
+ini_set("log_errors" , "0");
 ini_set("error_log" , "logs/error_log.log");
 ini_set("display_errors" , "0");
+error_reporting(0);
```
Comment (Csardelacal) [inline index.php:37]: I think I'd get rid of this. Since we do actually want the system to issue us warnings when we mess stuff up

**Review by Csardelacal — CHANGES_REQUESTED** [review]
I think the error reporting needs to be reset to match the server's config.

File: index.php (hunk 1 above)
Comment (pcgames3112) [reply index.php]: Removed it

**Review by Csardelacal — APPROVED** [review]
Perfect!

**Pull request merged**